lib/routes/gmu/namespace.ts

```diff
@@ -0,0 +1,9 @@
+import './news';
+import './yjs';
```

Comment:
Making it as resolved does not fix the issues. This is still an unsolved issue.

> Why import them?

_Originally posted by TonyRL in https://github.com/DIYgod/RSSHub/pull/19182#discussion_r2105850393_